tools/docker/hue/Dockerfile

```diff
@@ -85,7 +85,10 @@ RUN ./build/env/bin/pip install --no-cache-dir \
   # install after sqlalchemy-clickhouse and version == 1.0.4 \
   # otherwise Code: 516, Authentication failed will display \
   infi.clickhouse_orm==1.0.4 \
-  mysqlclient
+  mysqlclient \
+  PyAthena==2.25.2
```

Comment:
@pid998877 Shouldn't it be `pyathena` instead of `Pyathena`?

cf: https://pypi.org/project/pyathena/2.25.2/

Reply:
Hmm, maybe both works as pypi packages are case-sensitive I guess.